apps/console/src/extensions/configs/application.tsx

```diff
@@ -356,16 +356,12 @@ export const applicationConfig: ApplicationConfig = {
         },
         getStrongAuthenticationFlowTabIndex: (
             clientId: string,
-            tenantDomain: string,
-            templateId: string,
-            customApplicationTemplateId: string
+            tenantDomain: string
         ): number => {
             if (clientId === getTryItClientId(tenantDomain)) {
                 return 2; // For Asgardeo Try It App
-            } else if (templateId === customApplicationTemplateId) {
-                return 3; // For other apps built on Custom Application Templates
             } else {
-                return 4; // Anything else
+                return 3; // For other applications
```

Comment:
Shall we use constants for these values, otherwise it's hard to understand what these numbers are about at a first glance?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4591/commits/77bcc6d16d6548d6cb5d73c10f8d14433d601552